org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4whole/SimpleReporter.java

```diff
@@ -681,7 +724,12 @@ public void run(WorkerCallback out) throws Exception {
                         final Command cmd = cmds.get(i);
                         rep.tempfile = tempCNF;
                         cb(out, "bold", "Executing \"" + cmd + "\"\n");
-                        A4Solution ai = TranslateAlloyToKodkod.execute_commandFromBook(rep, world.getAllReachableSigs(), cmd, options);
+                        A4Solution ai = null;
+                        try { // [HASLab] this allows other commands to still be solved
```

Comment:
not Electrum related, but this allows "execute all" to finish all commands even when parsing error are thrown